src/css/embed.scss

```diff
@@ -114,6 +114,11 @@ $brexit-control-button-size: 28px;
     min-height: $baseline * 2;
     padding-bottom: $baseline * 1.5;
 }
+.brexit__content {
```

Comment:
This is used in the big number carousel. It is unaffected as all styles are overridden. Arguably this CSS needs a bit of TLC.